index.html

```diff
@@ -0,0 +1,127 @@
+            Our mission is to trained spread the good news about how Iot is redefining 
+            the future of tech</p>
+            <p class="bold">
+                <strong>2022.10.15(THU)~ 16(FRI)
+          </strong> </p>
+                 <p class="bottom-text">@ Accra International Conference Center</p>
+    </section>
+    <section class="section2">
+        <h2 class="h23">
+            Main Program
+        </h2>
+         <div class="underline"></div>
+
+            <div class="main-program"></div>
+                <ul class="blocks">
+                    <li class="b"><i class="C"><img src="img/1.png" alt=""></i>
+                        <p class="A">Seminars</p>
+                        <p class="B">we offer seminars to techs every two months
+                            per year. We attach importance to collabos
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/2.png" alt=""></i>
+                        <p class="A">Lectures</p>
+                        <p class="B">Learn mordern techniques
+                            in our daily classes from monday to friday
+                            each week
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/3.png" alt=""></i>
+                        <p class="A">Programs</p>
+                        <p class="B">Enrol and become eligible for our
+                            government sponsored susidies
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/4.png" alt=""></i>
+                        <p class="A">Assistance</p>
+                        <p class="B">Access technical support
+                            form our technical teams
+                            available to you 24 hours a day
+                        </p>
+                    </li>
+                    <li class="b"><i class="C"><img src="img/5.png" alt=""></i>
+                        <p class="A">Partners</p>
+                        <p class="B">Become acredited with us and gain
+                            the advantages that come with being an
+                            affiliate.
+                        </p>
+                    </li>
+                
+                </ul>
+                <button class="join" type="button">Attend The Summit</button>
+                <p class="psee" id="psee">SEE THE WHOLE PROGRAM</p>
+     </section>
+     <section class="speaker-sec">
+        <h2 class="speaker-header">Featured Guests</h2>
+        <div class="underline"></div>
+                <ul class="speaker-list" id="speaker-list">
+                   
+                </ul>
+            <button class="more-btn" id="more-btn" type="button">MORE<i class="fa fa-angle-down arrow"></i></button>
```

Comment:
- [ ] At the moment, your design does not have the background upper left checkbox image behind the speaker's image. Kindly include it so that it matches the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169511053-6ea5c2ce-bbb7-4079-8282-ecc36aaa7669.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/169510973-6e86493b-8cdc-4a2d-a8a2-668e9699358b.png)